Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -294,38 +295,52 @@ def demisto_types_to_xdr(_type: str) -> str:
         return xdr_type
 
 
-def _parse_demisto_comments(ioc: dict, comment_field_name: list[str] | str, comments_as_tags: bool) -> list[Any] | None:
+def create_an_indicator_link(ioc: dict) -> list[str]:
+    """
+    Creates an indicator link into comments field.
+    Args:
+        ioc (dict): the IOC dict.
+    Returns:
+        A list which contains a string of indicator's link.
+    """
+    if is_xsoar_saas():
+        return [f'{demisto.demistoUrls().get("server")}/indicator/{ioc.get("id")}']
+    return [f'{demisto.demistoUrls().get("server")}/#/indicator/{ioc.get("id")}']
+
+
+def _parse_demisto_comments(ioc: dict, comment_field_name: str, comments_as_tags: bool) -> list[Any] | None:
     """"
     Parsing xsoar fields to xdr from multiple fields value or a single value.
     Args:
         ioc (dict): the IOC dict.
-        comment_field_name (list[str] | str): the name of the comment field(s) to parse.
+        comment_field_name (str): the name of the comment field to parse.
         comments_as_tags (bool): whether to return comments as XDR tags rather than notes.
 
     Returns:
         A list with the parsed comment(s) joined by commas if multiple comment fields were provided,
         otherwise the parsed comment from the single provided field.
         Returns None if no comments were found.
     """
-    # parse comments from multiple fields if specified as list
-    if isinstance(comment_field_name, list):
-        comments = []
-        for field in comment_field_name:
-            parsing = parse_demisto_single_comments(ioc, field, comments_as_tags)
-            if parsing:
-                comments.extend(parsing)
-        return [', '.join(comments)]
 
-    # else return single field
-    return parse_demisto_single_comments(ioc, comment_field_name, comments_as_tags)
+    comments = []
+    # a regular comment for the given field name, "comments" as a default
+    single_comment_for_field_name = parse_demisto_single_comments(ioc, comment_field_name, comments_as_tags)
+    if single_comment_for_field_name:
+        comments.extend(single_comment_for_field_name)
+
+    # if the flag is True, add a link as a comment
+    if Client.add_link_as_a_comment:
+        comments.extend(create_an_indicator_link(ioc))
+
+    return (comments or ['']) if comments_as_tags else [', '.join(comments)]
```

Comment:
What happens when there is an indicator link? it is being attached as a tag?

Reply:
it's just a comment in XDR